app/main/views/audits.py

```diff
@@ -115,6 +115,27 @@ def list_audits():
     elif object_id:
         abort(400, 'object-id cannot be provided without object-type')
 
+    after_event_id = request.args.get('after-event-id')
```

Comment:
`per_page` and `earliest_for_each_object` are both underscores rather than dashes.  Might be better to go with underscores for this too?  Although `audit-type`, etc. are dashes, so we already have a mix...